examples/pod-gpc-example/src/gpcExample.ts

```diff
@@ -419,7 +419,7 @@ export async function gpcDemo(): Promise<boolean> {
       argumentType: ArgumentTypeName.String,
       value: serializeGPCProofConfig(pcdProofConfig)
     },
-    pod: {
+    [`${PODPCD_ARG_PREFIX}_pod0`]: {
```

Comment:
Use `podpcd_pod0` for clarity here.  I don't think we should make external devs use the prefix constant.

Also this is a tutorial, so I think the example should now include multiple PODs, and names friendlier than "pod0" so we can demonstrate how that works.  You could rework this section to use a similar multi-item proof to the one used for the non-PCD case above, though it can have a simpler config.  The goal of this section is to demonstrate what's unique to the GPCPCD layer, not all the possible features of GPCs.